tests/bofire/data_models/specs/surrogates.py

```diff
@@ -394,3 +396,36 @@
         "hyperconfig": None,
     },
 )
+
+specs.add_valid(
```

Comment:
Can you also add tests via `add_invalid` in which your custom validators for the `MultiTaskGPSurrogate` complain?